[BOT-X] Fix warning alert message

miguelcarreiro opened the pull request:
## Description
This pull request fixes warning alert message.

## Task items:
- [x] Added 'h' after warning times (start and end);
- [x] Added a new message when warning is already active.

miguelcarreiro committed:
fix: add 'h' after warning hour

miguelcarreiro committed:
fix: Add different tweet when warning already started

tiagojpdias commented on src/services/Warnings.js:
```diff
@@ -87,11 +87,16 @@ const getWarningsZones = (warningsZone, zone, client) => {
+    const strTime = (() => {
+      if (beginTime.isAfter(actualTime)) {
+        return `até às ${endTime.calendar(actualTime)} ${strEndDate}`;
+      } if (beginTime.isSame(endTime, 'day')) {
```
Put this in a new line.

```suggestion
      }
      
      if (beginTime.isSame(endTime, 'day')) {
```

miguelcarreiro committed:
fix: Add new line in services/Warnings

Improves code legibility

Co-Authored-By: Tiago Dias <tiagojpdias@gmail.com>

miguelcarreiro committed:
fix: Remove trailing spaces and improve code legibility

miguelcarreiro commented:
Since I've accepted @tiagojpdias suggestion, removed trailing spaces, and added a new line [after line 97](https://github.com/vostpt/bot/blob/50546d6c0cd69c9f1375384eda2f59e65cf20190/src/services/Warnings.js#L98), I think that PR can be merged into master.
If you think that PR shouldn't have been merged, please let me know.
